streams/src/test/java/org/apache/kafka/streams/integration/SuppressionIntegrationTest.java

```diff
@@ -519,113 +507,22 @@ private long scaledTime(final long unscaledTime) {
     }
 
     private void produceSynchronously(final String topic, final List<KeyValueTimestamp<String, String>> toProduce) {
-        final Properties producerConfig = mkProperties(mkMap(
-            mkEntry(ProducerConfig.CLIENT_ID_CONFIG, "anything"),
-            mkEntry(ProducerConfig.KEY_SERIALIZER_CLASS_CONFIG, STRING_SERIALIZER.getClass().getName()),
-            mkEntry(ProducerConfig.VALUE_SERIALIZER_CLASS_CONFIG, STRING_SERIALIZER.getClass().getName()),
-            mkEntry(ProducerConfig.BOOTSTRAP_SERVERS_CONFIG, CLUSTER.bootstrapServers())
-        ));
-        try (final Producer<String, String> producer = new KafkaProducer<>(producerConfig)) {
-            // TODO: test EOS
-            //noinspection ConstantConditions
-            if (false) {
-                producer.initTransactions();
-                producer.beginTransaction();
-            }
-            final LinkedList<Future<RecordMetadata>> futures = new LinkedList<>();
-            for (final KeyValueTimestamp<String, String> record : toProduce) {
-                final Future<RecordMetadata> f = producer.send(
-                    new ProducerRecord<>(topic, null, record.timestamp(), record.key(), record.value(), null)
-                );
-                futures.add(f);
-            }
-
-            // TODO: test EOS
-            //noinspection ConstantConditions
-            if (false) {
-                producer.commitTransaction();
-            } else {
-                producer.flush();
-            }
-
-            for (final Future<RecordMetadata> future : futures) {
-                try {
-                    future.get();
-                } catch (final InterruptedException | ExecutionException e) {
-                    throw new RuntimeException(e);
-                }
-            }
-        }
+        IntegrationTestUtils.produceSynchronously(CLUSTER.bootstrapServers(), topic, STRING_SERIALIZER, STRING_SERIALIZER, false, toProduce);
```

Comment:
Since I also needed this logic in the new integration test, I went ahead and added it to the integration test utils.